console/jsre/deps/doc.go

```diff
@@ -18,5 +18,18 @@
 // This file is derived from internal/jsre/deps/deps.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package deps contains the console JavaScript dependencies Go embedded.
+/*
+Package deps contains the console JavaScript dependencies Go embedded.
```

Comment:
I think the meaning of the sentence is not clear.

Reply:
@jeonghonah You're right. I changed the sentence. PTAL.